src/wwwroot/controllers/TemplateCtrl.js

```diff
@@ -25,22 +26,60 @@
 
     $scope.loadInProgress = false;
     $scope.saveInProgress = false;
+    $scope.isLegacyTemplate = true;
 
-    $scope.saveHtmlRaw = saveHtmlRaw;
+    $scope.save = save;
     $scope.saveAndEditWithMseditor = saveAndEditWithMseditor;
 
     if ($routeParams["templateId"]) {
       load($routeParams["templateId"]);
     }
+    else {
+      $scope.isLegacyTemplate = !$rootScope.forceMsEditor;
+    }
 
     /*************************/
     /* Function declarations */
     /*************************/
 
     function load(templateId) {
       $scope.loadInProgress = true;
-
-      return templates.getTemplateWithBody(templateId).then(function (result) {
+      return templates.getTemplate(templateId).then(function (template) {
+        switch (template.bodyType) {
+          case "empty":
+            $scope.isLegacyTemplate = false;
```

Comment:
When _bodyType_ is empty, the behavior should depends on _$rootScope.forceMsEditor_ value

Reply:
You are rigth, I forgot that, thanks andres.